templates/zerver/help/starring-messages.md

```diff
@@ -0,0 +1,23 @@
+#Starring a message
+**Starring messages is a good way to keep track of important messages, such as tasks you need to go back to,
+ or documents you reference often.**
+### How do I star a message?
+You can star a message on desktop clients by hovering over any message. The timestamp of the message should be replaced with three context buttons while hovering, one of which is a star. Click on this to star the message.
+
+![Message example](/static/images/help/star1.png)
+
+When a message is starred, a small green star appears next to the timestamp of the message.
+
+![Message example](/static/images/help/star2.png)
+
+### Unstarring messages
```

Comment:
Should match the style of previous heading. Probably you can simply say 'Unstar a message'.